model/atmosphere/diffusion/tests/diffusion/stencil_tests/test_apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence.py

```diff
@@ -28,9 +28,23 @@
 
 
 @pytest.mark.embedded_remap_error
+@pytest.mark.continuous_benchmarking
 class TestApplyDiffusionToWAndComputeHorizontalGradientsForTurbulence(StencilTest):
     PROGRAM = apply_diffusion_to_w_and_compute_horizontal_gradients_for_turbulence
     OUTPUTS = ("w", "dwdx", "dwdy")
+    STATIC_PARAMS = {
+        StandardStaticVariants.NONE: None,
+        StandardStaticVariants.COMPILE_TIME_DOMAIN: (
+            "horizontal_start",
+            "horizontal_end",
+            "vertical_start",
+            "vertical_end",
+        ),
+        StandardStaticVariants.COMPILE_TIME_VERTICAL: (
+            "vertical_start",
+            "vertical_end",
+        ),
```

Comment:
```suggestion
            "vertical_end",
            "limited_area",
        ),
        StandardStaticVariants.COMPILE_TIME_VERTICAL: (
            "limited_area",
            "vertical_start",
            "vertical_end",
        ),
```

Reply:
This seems to be leading to test failures in the unit tests. Are you shure about this?